Traceback (most recent call last):
  File "/home/ubuntu/AI-Study/week7/basic/train.py", line 54, in <module>
    args, training_args = parser.parse_args_into_dataclasses()
                          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/ubuntu/AI-Study/week7/basic/.venv/lib/python3.12/site-packages/transformers/hf_argparser.py", line 358, in parse_args_into_dataclasses
    obj = dtype(**inputs)
          ^^^^^^^^^^^^^^^
  File "<string>", line 132, in __init__
  File "/home/ubuntu/AI-Study/week7/basic/.venv/lib/python3.12/site-packages/transformers/training_args.py", line 1761, in __post_init__
    self.device
  File "/home/ubuntu/AI-Study/week7/basic/.venv/lib/python3.12/site-packages/transformers/training_args.py", line 2297, in device
    return self._setup_devices
           ^^^^^^^^^^^^^^^^^^^
  File "/home/ubuntu/AI-Study/week7/basic/.venv/lib/python3.12/site-packages/transformers/utils/generic.py", line 67, in __get__
    cached = self.fget(obj)
             ^^^^^^^^^^^^^^
  File "/home/ubuntu/AI-Study/week7/basic/.venv/lib/python3.12/site-packages/transformers/training_args.py", line 2167, in _setup_devices
    raise ImportError(
ImportError: Using the `Trainer` with `PyTorch` requires `accelerate>=0.26.0`: Please run `pip install transformers[torch]` or `pip install 'accelerate>=0.26.0'`
